File: 문제풀이/온라인 저지/백준/# 01. Bronze Ⅴ/# 01008. A／B/java/Main.java
import java.io.IOException;

public class Main {

    public static void main(String[] args) throws IOException {
        int a = System.in.read() - 48;
        System.in.read();
        double b = System.in.read() - 48;
        System.out.print(a/b);
    }

}

File: 문제풀이/온라인 저지/백준/# 01. Bronze Ⅴ/# 01008. A／B/kotlin/Main.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val st = StringTokenizer(br.readLine())
    print(st.nextToken().toInt() / st.nextToken().toDouble())
    br.close()
}
